debian/changelog

```diff
@@ -1,3 +1,24 @@
+adsys (0.10.1) lunar; urgency=medium
+
+  [ Denison Barbosa ]
+  [ Jean-Baptiste Lallement ]
+  [ Gabriel Nagy ]
+  [ Didier Roche ]
+  * Fix erroneous non alternative dependency on package krb5-user
+  * Fix a bug in internal/config tests that was causing the autopkgtests to fail
+  * Update internal/config to also trigger a reload when config file is
+    overwritten
```

Comment:
Was I to vague or to specific? I have no idea if I should describe more of the problem

Reply:
`git blame` disagrees 🤣